miner/stress/main.go

```diff
@@ -65,9 +66,10 @@ func main() {
 	signal.Notify(interruptCh, os.Interrupt)
 
 	var (
-		stacks []*node.Node
-		nodes  []*eth.Ethereum
-		enodes []*enode.Node
+		stacks  []*node.Node
+		nodes   []*eth.Ethereum
+		enodes  []*enode.Node
+		beacons []*catalyst.SimulatedBeacon
```

Comment:
It doesn't look like this is used anywhere?

Reply:
fixed in https://github.com/MariusVanDerWijden/go-ethereum/pull/47/files